Make dist is broken

### Steps

1. `git clean -fdx`
1. `make dist`

### Expected
:tada: 

### Actual
:boom: 
```
% make dist
make appstore
make[1]: Entering directory '/home/vincent/Private/Work/workspace/owncloud-for-signing/apps/guests'
rm -rf /home/vincent/Private/Work/workspace/owncloud-for-signing/apps/guests/build/dist
mkdir -p /home/vincent/Private/Work/workspace/owncloud-for-signing/apps/guests/build/dist/guests
cp -R appinfo controller js l10n lib settings templates LICENSE README.md CHANGELOG.md /home/vincent/Private/Work/workspace/owncloud-for-signing/apps/guests/build/dist/guests
cp: cannot stat 'controller': No such file or directory
cp: cannot stat 'settings': No such file or directory
make[1]: *** [Makefile:88: appstore] Error 1
make[1]: Leaving directory '/home/vincent/Private/Work/workspace/owncloud-for-signing/apps/guests'
make: *** [Makefile:81: dist] Error 2
```

@VicDeo I remember you PSR'd all the things ? can you adjust ?

--- a/Makefile
+++ b/Makefile
@@ -14,7 +14,7 @@ JSDOC=$(NODE_PREFIX)/node_modules/.bin/jsdoc
 
 app_name=$(notdir $(CURDIR))
 doc_files=LICENSE README.md CHANGELOG.md
-src_dirs=appinfo controller js l10n lib settings templates
+src_dirs=appinfo js l10n lib templates
 all_src=$(src_dirs) $(doc_files)
 build_dir=$(CURDIR)/build
 dist_dir=$(build_dir)/dist

